pandas/tests/arrays/test_datetimelike.py

```diff
@@ -47,12 +46,7 @@ def period_index(freqstr):
     the PeriodIndex behavior.
     """
     # TODO: non-monotone indexes; NaTs, different start dates
-    with warnings.catch_warnings():
```

Comment:
What makes this redundant? We may want to keep this just in case another test uses this fixture with "B"

Reply:
@natmokval i was wondering this too

Reply:
Thank you for the comments and sorry for the late reply.
Yes, I agree, some test may need to use the fixture `period_index` with “B”.
So far we only use this fixture in class `TestPeriodArray` and we also use `filterwarning`. I think it’s enough to use
```
@pytest.mark.filterwarnings(r"ignore:Period with BDay freq is deprecated:FutureWarning”)
```
to suppress this warning, we don’t need to do it in the fixture. Am I missing something?

Reply:
agreed, I close this PR.